Examples/Tests/magnetostatic_eb/analysis_rz.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
This is the benchmark script for the Python RZ CI test, which doesn't check particles because of the random_theta issue I am  having and the non-reproducible random numbers from my platform to the Ubuntu container on Azure.